I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 184, in script
    if train2_srting and string_number > 0:
  File "game/script.rpy", line 184, in <module>
    if train2_srting and string_number > 0:
NameError: name 'train2_srting' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 184, in script
    if train2_srting and string_number > 0:
  File "renpy/ast.py", line 1892, in execute
    if renpy.python.py_eval(condition):
  File "renpy/python.py", line 2249, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "renpy/python.py", line 2242, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 184, in <module>
    if train2_srting and string_number > 0:
NameError: name 'train2_srting' is not defined

Windows-10-10.0.19041
Ren'Py 7.4.6.1693
CGJ2021 1.0
Sat Jul 10 21:16:11 2021
